examples/dingo-gov-lens/static/styles.css

```
@@ -0,0 +1,436 @@
+  font-size: 12px;
+  letter-spacing: 0;
+  text-transform: uppercase;
+}
+
+tbody tr:hover {
+  background: rgba(66, 133, 244, 0.12);
+}
+
+.mono {
+  font-family:
+    "SFMono-Regular", Consolas, "Liberation Mono", Menlo, monospace;
```

Stylelint (17.12.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)